warp10/src/main/java/io/warp10/continuum/gts/GTSHelper.java

```diff
@@ -2338,6 +2338,11 @@ public static final GeoTimeSerie bucketize(GeoTimeSerie gts, long bucketspan, in
       } else if (i < 0) {
```

Comment:
also, on line 2231, in the most usual case you can save the call to binary search by using >= instead of >

Reply:
indeed for line 2331, but what is the problem with line 2338 ?

Reply:
The editor didn't allowed commenting on line 2331 so I add the comment line 2338